src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -389,13 +393,15 @@ def voronoi_cells(V):
       of ``E``) with identical first and last elements)
     - ``DG`` -- the dual graph of ``V``, where the vertices are labelled
       by the compact regions of ``V`` and the edges by their dual edges.
+    - ``vertical_regions`` -- dictionary for the regions associated
+      with vertical lines.
```

Comment:
```suggestion
      with vertical lines
```
We don't end `INPUT:`'s with a period (the one just above is wrong and should be fixed).